graph.py

```diff
@@ -179,7 +179,19 @@ def __init__(self, channel, arch, constraints):
         self.constrain_graph(
             self.raw.graph, self.noarch.graph, self.package_constraints
         )
-        self.cache = TTLCache(100, ttl=600)
+        self.ttl = ttl
+        self._cache = TTLCache(100, ttl=ttl)
```

Comment:
can we move this to a class attr?

Reply:
sure

Reply:
There already is a class atter called `_cache`